app/models/declaration.rb

```diff
@@ -4,6 +4,18 @@ class Declaration < ApplicationRecord
   UPLIFT_PAID_STATES = %w[paid awaiting_clawback clawed_back].freeze
   COURSE_IDENTIFIERS_INELIGIBLE_FOR_UPLIFT = %w[npq-additional-support-offer npq-early-headship-coaching-offer].freeze
   VOIDABLE_STATES = %w[submitted eligible payable ineligible].freeze
+  QUALIFICATION_TYPES = {
+    "npq-leading-teaching" => "NPQLT",
+    "npq-leading-behaviour-culture" => "NPQLBC",
+    "npq-leading-teaching-development" => "NPQLTD",
+    "npq-leading-literacy" => "NPQLL",
+    "npq-senior-leadership" => "NPQSL",
+    "npq-headship" => "NPQH",
+    "npq-executive-leadership" => "NPQEL",
+    "npq-early-years-leadership" => "NPQEYL",
+    "npq-leading-primary-mathematics" => "NPQLPM",
+    "npq-senco" => "NPQSENCO",
+  }.freeze
```

Comment:
Should we update `config/initializers/courses.rb` with the short codes instead of having it here?